tests/threads/tthread_destroy_before_finish.nim

```diff
@@ -0,0 +1,27 @@
+discard """
+  output: ""
+  joinable: false
+  description: '''
+  . if `Thread` is freed before the thread finishes, SIGSEGV
+  . reference counting of `Thread.coreFn` is broken
+'''
```

Comment:
```suggestion
  knownIssue: '''
  . if `Thread` is freed before the thread finishes, SIGSEGV
  . reference counting of `Thread.coreFn` is broken
'''
```

That should mark this tests as a known issue and it won't fail in CI.

Reply:
Don't you think `knowIssue: true` is better?

Reply:
can take the reason directly to afford a literal style. The test description as written is more a failure description. I'm not super fussed either way, using `knownIssue` seems like the right tact, the exact details aren't super important.